### PR TITLE
Dotted function for top border

### PR DESCRIPTION
**Pull request by PayalGodse**
Added call to dotted function

### CODE REVIEW

**Comment by hrj** [comment]
@PayalGodse There are too many changes due to formatting differences. My bad; I should have told you earlier.

I have added a new section called [Coding](https://github.com/UprootLabs/gngr/wiki/Developer:Setup#coding) in Dev Setup.

Please,
* setup the formatter as described in the wiki page.
* format BaseElementRenderable.java
* commit the changes and push to the same branch (this PR will update automatically)

File: src/HTML_Renderer/org/lobobrowser/html/renderer/BaseElementRenderable.java [hunk 1]
```diff
@@ -152,17 +156,65 @@ protected void invalidateLayoutLocal() {
...
-      final int dwInt = this.getDeclaredWidthImpl(renderState, actualAvailWidth);
+      final int dwInt = this.getDeclaredWidthImpl(renderState,
+          actualAvailWidth);
       dw = dwInt == -1 ? null : new Integer(dwInt);
       this.declaredWidth = dw;
     }
     return dw;
   }
 
+  protected Integer getDeclaredHelper(final RenderState renderState,
```
Comment (hrj) [inline src/HTML_Renderer/org/lobobrowser/html/renderer/BaseElementRenderable.java]: This and the following four functions (getDeclared*) seem to have been newly added but not used. Looks like a copy-paste error. Can you remove them and verify there are no compilation errors?

**Comment by hrj** [comment]
Oh wait, I see the problem. This PR is trying to merge into v000 branch. It needs to merge into master branch.

I think you should start afresh. Create a new branch based on this repository's master. Then copy your changes into the respective files and commit. Then create a new PR against the master branch.

**Comment by PayalGodse** [comment]
Ohh okay..Will do it!
I apologise for the confusion created!

Best,
Payal

**Pull request closed without merging**